src/Features/Core/Portable/Completion/SuggestionMode/SuggestionModeCompletionProvider.cs

```diff
@@ -15,16 +15,8 @@ internal abstract class SuggestionModeCompletionProvider : CommonCompletionProvi
 
         public override async Task ProvideCompletionsAsync(CompletionContext context)
         {
-            if (context.Options.GetOption(CompletionControllerOptions.AlwaysShowBuilder))
-            {
-                var text = await context.Document.GetTextAsync(context.CancellationToken).ConfigureAwait(false);
-                context.SuggestionModeItem = this.CreateEmptySuggestionModeItem();
-            }
```

Comment:
@Pilchie It was already dead--we weren't actually setting that option any where. The controller/model were already handling the "empty builder" case.